specs/BrowserTaskManager.md

```diff
@@ -0,0 +1,69 @@
+# Background
+The WebView2 team has been asked to provide an API to expose the Browser Task
+Manager. At the current time, it is expected that the user will close the Task 
+Manager manually, so we do not need to provide an API to close it.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+We propose extending `CoreWebView2` to provide a `ShowTaskManager` method.
```

Comment:
On the declaration + impl PR, Liang had discussed the name for this method. We were floating between a few options:

- `LaunchTaskManager`
- `OpenTaskManager`
- `ShowTaskManager`

as well as possibly adding "Browser" before task manager in the above options.

The actual implementation calls `chrome::ShowTaskManager`, which is why I chose to name it `ShowTaskManager` for now, but I also feel like `OpenTaskManager` may make more sense as we are opening a new window.